src/Components/People.js

```diff
@@ -0,0 +1,72 @@
+import axios from "axios";
+import { Component } from "react";
+
+class People extends Component {
+  state = { input: "", person: {}, name: "", age: "", gender: "", show: false };
+
+  handleInput = (e) => {
+    this.setState({ input: e.target.value });
+  };
+
+  hanldeSubmit = async (e) => {
+    e.preventDefault();
+    this.setState({ show: true });
+    try {
+      const { input } = this.state;
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/people`);
+      debugger;
+      const ghibilPerson = res.data.filter((person) => person.name === input);
```

Comment:
Great use of filter. If you add "[0]" to the end of this line you won't have to add 0 on every line below